packages/nexus-sdk/src/File/index.ts

```diff
@@ -1,22 +1,192 @@
-import { toPromise } from '@bbp/nexus-link';
+import FormData from 'isomorphic-form-data';
```

Comment:
I want to avoid having that kind of stuff as much as possible in our code, this is when things start getting messy...